app/_locales/en/messages.json

```diff
@@ -665,6 +665,9 @@
   "editGasMaxFeeLow": {
     "message": "Max fee too low for network conditions"
   },
+  "editGasMaxFeePriorityImbalance": {
+    "message": "Max fee must be larger than max priority fee"
```

Comment:
Perhaps this should say `"Max fee cannot be lower than max priority fee"`, because it can be equal, not just "larger"